packages/cogify/src/cogify/cli/cli.cover.ts

```diff
@@ -23,6 +23,17 @@ export function gsdToMeter(gsd: number): number {
   return parseFloat(gsd.toFixed(3));
 }
 
+// parse background color from string "r,g,b,a"
+function parseBackgroud(background: string): Background {
```

Comment:
could we reuse one of our other parseRgba functions here, we mostly use the browser based, '#aa00ff00' in other spaces, do we need another format?